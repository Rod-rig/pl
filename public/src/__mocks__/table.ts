export default {
  table: [
    {
      'd': 3,
      'goals against': 21,
      'goals for': 88,
      'l': 1,
      'points': 84,
      'position': 1,
      'teamName': 'Manchester City',
      'w': 27,
    },
    {
      'd': 5,
      'goals against': 23,
      'goals for': 60,
      'l': 5,
      'points': 68,
      'position': 2,
      'teamName': 'Manchester United',
      'w': 21,
    },
    {
      'd': 9,
      'goals against': 35,
      'goals for': 75,
      'l': 4,
      'points': 66,
      'position': 3,
      'teamName': 'Liverpool',
      'w': 19,
    },
  ],
};
