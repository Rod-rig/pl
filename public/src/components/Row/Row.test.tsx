import {Table} from '@material-ui/core';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import table from '../../__mocks__/table';
import {Row} from './';

describe('Row', () => {
  const team = table.table[0];
  const chars = Object.keys(team);

  it('should render correctly', () => {
    const tree = renderer
      .create(<Table><Row row={team} chars={chars}/></Table>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
