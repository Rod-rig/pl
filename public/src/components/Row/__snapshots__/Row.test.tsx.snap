// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render correctly 1`] = `
<tr
  className="MuiTableRow-root-1"
>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    1.
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    Манчестер Сити
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    24
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    21
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    2
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    1
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    70:18
  </td>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6 MuiTableCell-paddingDefault-11"
    scope={undefined}
  >
    65
  </td>
</tr>
`;
